src/devtools/client/webconsole/utils/autocomplete-eager.ts

```diff
@@ -64,9 +65,13 @@ async function eagerEvaluateExpression(
 
 export function useEagerEvaluateExpression() {
   const frame = useSelector(getSelectedFrame);
-  const callback = useCallback(
-    (expression: string) =>
-      frame ? eagerEvaluateExpression(expression, frame.asyncIndex, frame.protocolId) : null,
+  const callback = useMemo(
+    () => (expression: string) => {
+      if (frame) {
```

Comment:
This is the tiniest nit in the entire world, but I think I would reorder these to make the check read like a guard clause:

```
if (!frame) {
  return null;
}

return eagerEvaluateExpression(expression, frame.asyncIndex, frame.protocolId);
```

Either way is fine, I just feel like conforming to really common patterns like guard clauses can make the cognitive load a little lighter while reading.